inventory/commands/commands.go

```diff
@@ -0,0 +1,62 @@
+package commands
```

Comment:
github branching strategy should be maintained
main/master should have reviewed code only